README.md

```diff
@@ -1,4 +1,5 @@
 # ElixirAgi
+[![Build Status](https://travis-ci.org/alisinabh/elixir_agi.svg?branch=master)](https://travis-ci.org/alisinabh/elixir_agi)
```

Comment:
This doesn't belong to this pull, please open a different pull request if needed.